malariagen_data/anoph/h12.py

```diff
@@ -514,6 +514,269 @@ def plot_h12_gwss(
         else:
             return fig
 
+    @check_types
+    @doc(
+        summary="Plot h12 GWSS data track with multiple traces.",
+    )
+    def plot_h12_gwss_track_multi(
+        self,
+        contig: base_params.contig,
+        sample_queries: h12_params.sample_queries,
```

Comment:
```suggestion
        cohorts: base_params.cohorts,
```

Suggest to use a "cohorts" parameter instead, which is the same as already used elsewhere in the API when multiple cohorts are required. 

This could then be handled via the `_setup_cohort_queries()` helper function, like is done within the `diversity_stats()` method.

Reply:
Also suggest to add a `sample_query` parameter, with behaviour as per `diversity_stats()`. I.e., the `sample_query` defines a base query, then the `cohorts` defines groups of samples on top of that. Should all be handled via the `_setup_cohort_queries()` helper.